src/utils/hints/nftsHints.js

```diff
@@ -1,7 +1,23 @@
 export const nftsHints = {
```

Comment:
There were 2 files for NFT hints. Probably product of some merging. I united it into just one. Hence every nft hint should also be checked